### PR TITLE
Issue 1 user can join waitlist - COMPLETE

### PR DESCRIPTION
**Pull request by j-alejandro-araujo**
Screenshot 2023-06-18 at 10 52 49 AM
Screenshot 2023-06-18 at 10 52 29 AM
Screenshot 2023-06-18 at 10 52 22 AM

### CODE REVIEW

File: css/layout.css [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+.desktop-nav {
+  display: none;
+}
+
+@media screen and (max-width: 768px) {
+  /* #about-us-text {
+    margin: 0;
+    padding: 0;
+  } */
```
Comment (shawnkost) [inline css/layout.css:13]: Remove all old code

File: index.html [hunk 2]
```diff
@@ -8,12 +8,139 @@
...
+        <div id="popup">
+          <div class="modal">
+            <div id="modal-title">
+              <p id="modal-text">Sign Up</p>
+            </div>
+            <div id="modal-inputs-container">
+              <label for="name"></label>
+              <input type="text" id="name" name="name" placeholder="Name" required>
+              <label for="email"></label>
+              <input type="email" id="email" placeholder="Email" required>
```
Comment (shawnkost) [inline index.html:31]: If you don't have any texts inside the `label`s, then they aren't needed. We only need to include a `label` when we have some sort of text associated wth the `input`s

File: js/main.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  if (window.innerWidth <= 768) {
+    $aboutImage.style.display = 'none';
+  } else {
+    $aboutImage.style.display = 'flex';
+  }
+}
+
+document.addEventListener('DOMContentLoaded', function () {
+  hideImage();
+});
```
Comment (shawnkost) [inline js/main.js:53]: You already have a `DOMContentLoaded` event listener, just move the `hideImage` function call into the other event.

**Review by shawnkost — CHANGES_REQUESTED** [review]
Some changes here Jesus! I also sent you a ping regarding your `h1` elements. Depending on your response, that might need some adjustment as well

**Comment by j-alejandro-araujo** [comment]
Adjusted code per the flagged comments!

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**